common/scala/src/main/scala/whisk/core/containerpool/HttpUtils.scala

```diff
@@ -141,14 +166,15 @@ protected[core] class HttpUtils(hostname: String, timeout: FiniteDuration, maxRe
 object HttpUtils {
 
   /** A helper method to post one single request to a connection. Used for container tests. */
-  def post(host: String, port: Int, endPoint: String, content: JsValue): (Int, Option[JsObject]) = {
+  def post(host: String, port: Int, endPoint: String, content: JsValue)(implicit logging: Logging,
+                                                                        tid: TransactionId): (Int, Option[JsObject]) = {
     val connection = new HttpUtils(s"$host:$port", 90.seconds, 1.MB)
     val response = connection.post(endPoint, content, retry = true)
     connection.close()
     response match {
       case Right(r)                   => (r.statusCode, Try(r.entity.parseJson.asJsObject).toOption)
       case Left(NoResponseReceived()) => throw new IllegalStateException("no response from container")
-      case Left(Timeout())            => throw new java.util.concurrent.TimeoutException()
+      case Left(Timeout(t))           => throw new java.util.concurrent.TimeoutException()
```

Comment:
Could drop the value (i.e. use `Timeout(_)`)

Reply:
Good catch. Done.